backend/controllers/whereabouts/scheduledMoves.ts

```diff
@@ -58,6 +72,39 @@ export default ({ prisonApi, offenderSearchApi }) => {
         classes: alert.classes,
       }))
   }
+
+  const getHoldAgainstTransferAlertDetails = async (
+    context,
+    prisonerDetails: PrisonerSearchResult
+  ): Promise<HoldAgainstTransferAlerts> => {
+    const activeHoldAgainstTransferAlerts = prisonerDetails.alerts.filter((alert) =>
+      relevantAlertsForHoldAgainstTransfer.includes(alert.alertCode)
+    )
+
+    if (activeHoldAgainstTransferAlerts.length === 0) {
+      return null
+    }
+
+    const activeHoldAgainstTransferAlertDetails = await prisonApi.getAlertsForLatestBooking(context, {
+      offenderNo: prisonerDetails.prisonerNumber,
+      alertCodes: relevantAlertsForHoldAgainstTransfer,
+      sortBy: 'dateCreated',
+      sortDirection: 'DESC',
+    })
+
+    return {
+      prisonerNumber: prisonerDetails.prisonerNumber,
+      displayId: uuidv4(),
```

Comment:
I tried to do this in the NJK, but couldn't get it working. I have found an npm package that might do the trick...